### DISCUSSION TITLE
Dragonfly Incompatible with Bull?

### DISCUSSION THREAD

**Discussion by BlackYuzia** [dragonflydb/dragonfly#781] [Q&A]
Hi there.
When I try use `Dragonfly` with [`Nest Bull`](https://docs.nestjs.com/techniques/queues) I get error:
![image](https://user-images.githubusercontent.com/25010528/218322802-c547ecd3-a203-4e89-9ab2-369ea5cb4007.png)

I can fix this by settings or something? When I try `Redis` this code work's fine.
![image](https://user-images.githubusercontent.com/25010528/218322870-67937ba2-06eb-4714-bfc2-5f9d97903c83.png)

**Comment by romange** [comment]
Hi @BlackYuzia , could you write a minimal app that reproduces it?

**Reply by BlackYuzia** [reply]
@romange 

[`Repo`](https://github.com/BlackYuzia/Dragonfly-Bull)
For change host of redis/dragonfly edit `src/app.module.ts`.

For run app: `npm run start:dev`

**Reply by romange** [reply]
Thanks, @BlackYuzia. I built it. how do I trigger the handler? or how to I reach the error state?

**Reply by romange** [reply]
ah I see, it happens immediately.

**Reply by romange** [reply]
Filed #782

**Reply by DanielNetzer** [reply]
for followup reference https://github.com/taskforcesh/bullmq/discussions/1689

**Comment by filip3f** [comment]
- Check for compatibility issues: Make sure that the versions of both packages you're using are compatible with each other. You can check the documentation or release notes for each package to find out which versions are compatible.
- Check for dependencies: Make sure that any dependencies required by both packages are installed and up to date. You can use pip, the package manager for Python, to install and update dependencies.
- Check for conflicting libraries: It's possible that the packages you're using require different versions of the same library, causing conflicts. Try uninstalling and reinstalling both packages, or using a virtual environment to isolate the packages from each other.
